### ISSUE TITLE
How to obtain checkpoint?

### ISSUE THREAD

**Issue by HLinChen** [SAITPublic/rascanet#1]
Thanks for the code! However, when I want to run the model, I can't find ckeckpoint files. How can I obtain them? Also, if I try to reproduce results in the paper, how to set hyperparameters, like learning rate, batch size, weight decay, epochs etc? Looking forward to receiving your reply!

**Comment by jakeyoo87** [comment]
Hello, HLinChen, I found out that checkpoints were not included due to the .gitignore files. I will upload it soon (It takes some time since I have to go under some company policies). For the hyperparameter, we used AdamW, wd=1e-5, batchsize=128, lr=2e-3 , epoch=200 with CosineScheduler and (linear warmup=80).

**Comment by jakeyoo87** [comment]
Hello, I have re-uploaded a checkpoints !
